2.5.x ES and Graphite HTTP dashboards.

henrikengstrom opened the pull request.

henrikengstrom committed:
2.5.x ES and Graphite HTTP dashboards.

pvlugter commented on cinnamon/2.5.x/grafana/4.x/elasticsearch/http-client-metrics.json:
```diff
@@ -0,0 +1,606 @@
+{
+  "dashboard": {
```
Dashboards in this repo shouldn't have the "dashboard" wrapper so that they can be loaded from the Grafana load json UI. Was loading this tested?

pvlugter commented on cinnamon/2.5.x/grafana/4.x/graphite/http-client-metrics.json:
```diff
@@ -0,0 +1,343 @@
+    {
+      "type": "query",
+      "label": null,
+      "query": "*.gauges.servers.*.apps.*.metrics.akka-http.systems.*.client-pools.*",
```
Shouldn't we have server and app variables as well?

henrikengstrom replied:
I can add that.

pvlugter commented on cinnamon/2.5.x/grafana/4.x/graphite/http-client-metrics.json:
```diff
@@ -0,0 +1,343 @@
+        "steppedLine": false,
+        "targets": [
+          {
+            "target": "aliasByNode(*.gauges.servers.*.apps.*.metrics.akka-http.systems.*.client-pools.*.connections, 11)",
```
Should there be an actor system variable as well?

henrikengstrom replied:
Sure. Will add.

pvlugter commented on cinnamon/2.5.x/grafana/4.x/graphite/http-client-metrics.json:
```diff
@@ -0,0 +1,343 @@
+      },
+      {
+        "id": 3,
+        "title": "$Services - Requests: 1min rate",
```
Is this title correct? The targets show service response time.

pvlugter commented on cinnamon/2.5.x/grafana/4.x/graphite/http-server-metrics.json:
```diff
@@ -0,0 +1,878 @@
+              "textEditor": true
+            },
+            {
+              "target": "aliasByNode(stats.gauges.servers.*.apps.*.metrics.akka-http.systems.*.http-servers.$HttpServers.connections.min15_rate, 11, 12, 13)",
```
Should we still have min1, min5, and min15 rates in the same graph?

henrikengstrom replied:
I will remove.

pvlugter commented on cinnamon/2.5.x/grafana/4.x/graphite/http-server-metrics.json:
```diff
@@ -0,0 +1,878 @@
+              "textEditor": true
+            },
+            {
+              "target": "aliasByNode(stats.gauges.servers.*.apps.*.metrics.akka-http.systems.*.http-servers.$HttpServers.responses-2xx.min15_rate, 11, 13)",
```
Also here: graph has min1, min5, and min15 rates?

pvlugter reviewed (changes requested):
I haven't tried these out yet, but noticed some things when skimming through the json.

Also, have we checked that we have 10s intervals set. We've had those issues with sawtooth graphs when we haven't had that.

pvlugter committed:
Tweak akka http elasticsearch dashboards

+ use multi select for host, application, and actor system
+ fix missing "application" field for response time 2xx
+ fix incorrect alias for endpoint rates (remove -max)
+ use min of mins aggregation, rather than max of mins

pvlugter committed:
Tweak akka http graphite dashboards

+ use multi select for host, application, and actor system

pvlugter commented:
Added some small tweaks and fixes to the dashboards.
